### ISSUE TITLE
Is it ok with IntelliJ 16.x?

### ISSUE THREAD

**Issue by seovchinnikov** [shyiko/native2asciiplug#4]
Hey!
Just tried plugin with IDEA 16.1.4 and it seems like it does not work.
Is it really to get it work with it or IDE has a lot of changes in its core since last commit?
Thanks

**Comment by shyiko** [comment]
Hi @seovchinnikov. AFAIK, there is no need for this plugin starting from IDEA 13 (I think). Go to "Settings -> Editor -> File Encodings". You should see default encoding + an option to apply "transparent native-to-ascii conversion" in flight.

**Comment by seovchinnikov** [comment]
@shyiko As I know, IDEA's option can be applied for in-place conversion only.  But if I understand it right main idea of this (and its ancestor) plugin is to convert only target version of .properies. It can be very useful in some cases. Am I right?

**Comment by shyiko** [comment]
You're right. IDEA does conversion in place. I see how it could be useful (target instead of source conversion) but compiler API (IDEA's API in general) proved to be too unstable to maintain this project. I can grant you committer access if you're interested, though.
